schemas/update-labware/1-update-labware.avsc

```diff
@@ -0,0 +1,69 @@
+{
+  "namespace": "uk.ac.sanger.psd",
+  "type": "record",
+  "name": "UpdateLabwareMessage",
+  "doc": "A message describing the field level updates needed to be applied to a labware.",
+  "fields": [
+    {
+      "name": "messageUuid",
+      "doc": "Unique message ID.",
+      "type": {"name": "version4MessageUuid", "type": "fixed", "size": 36}
+    },
+    {
+      "name": "messageCreateDateUtc",
+      "doc": "Date (UTC) that the message was created.",
+      "type": {
+        "type": "long",
+        "logicalType": "timestamp-millis"
+      }
+    },
+    {
+      "name": "labwareUpdates",
+      "doc": "Labware object data.",
+      "type": [
+        "null",
+        {
+          "type": "array",
+          "items": {
+            "name": "Labware", "type": "record",
+            "doc": "Labware data.",
+            "fields": [
+              {
+                "name": "labwareUuid",
+                "doc": "Unique labware ID, assigned by PAM.",
+                "type": {"name": "version4LabwareUuid", "type": "fixed", "size": 36}
+              },
+              { "name": "name", "type": "string"},
+              { "name": "value", "type": "string" }
```

Comment:
These last two fields are missing `doc `entries.  I think they're borderline, but it might be a good idea to be explicit about what a `name` is and what `value` is used for.

Reply:
ok